src/launcher/util/appCompatibilityWarning.tsx

```diff
@@ -104,17 +111,96 @@ const checkMinimalRequiredAppVersions: AppCompatibilityChecker = app => {
           );
 };
 
-export default (
+const nrfutilDeviceToJLink = (device: string) => {
+    if (semver.gte(device, '1.4.4') && semver.lt(device, '2.0.0')) {
+        return 'V7.80c';
+    }
+
+    if (semver.gte(device, '2.0.0') && semver.lt(device, '2.1.0')) {
+        return 'V7.88j';
+    }
+
+    if (semver.gte(device, '2.1.0') && semver.lt(device, '2.5.44')) {
+        return 'V7.94e';
+    }
+
+    if (semver.gte(device, '2.5.4')) {
+        return 'V7.94i';
+    }
+};
+
+const checkJLinkRequierments: AppCompatibilityChecker = async (
+    app: LaunchableApp
+) => {
+    if (!isDownloadable(app)) {
+        return undecided;
+    }
+
+    const deviceVerison =
+        app.versions?.[app.currentVersion]?.nrfutilModules?.device?.at(0);
+
+    if (!deviceVerison) {
+        return undecided;
+    }
+
+    const requiredVerion = nrfutilDeviceToJLink(deviceVerison);
+
+    const userDir = getUserDataDir();
+    const sandbox = await getSandbox(userDir, 'device', deviceVerison);
+    const moduleVersion = await sandbox.getModuleVersion();
+    const jlinkVersion = resolveModuleVersion(
+        'JlinkARM',
+        moduleVersion.dependencies
```

Comment:
Review all the spelling for `version`.
```suggestion
    const deviceVersion =
        app.versions?.[app.currentVersion]?.nrfutilModules?.device?.at(0);

    if (!deviceVersion) {
        return undecided;
    }

    const requiredVersion = nrfutilDeviceToJLink(deviceVersion);

    const userDir = getUserDataDir();
    const sandbox = await getSandbox(userDir, 'device', deviceVersion);
    const moduleVersion = await sandbox.getModuleVersion();
    const jlinkVersion = resolveModuleVersion(
        'JlinkARM',
        moduleVersion.dependencies
```